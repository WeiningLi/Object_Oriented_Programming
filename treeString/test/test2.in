ra ""
^a6 "the greatest (?) "
pa
rb ", how are you?"
pb
+aab
pa
ia1?
ia8d
pa
da
rb "im fine"
+aab
rb "actually not, I cannot finish my assns, and no one can help"
ccb
+aac
da
pa
q
