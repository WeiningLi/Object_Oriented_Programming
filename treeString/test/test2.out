the greatest (?) 
, how are you?
the greatest (?) , how are you?
t?e greadest (?) , how are you?
"t?e greadest (?) " size: 17
 ", how are you?" size: 14
"im fine" size: 38
 ", how are you?" size: 31
  "t?e greadest (?) " size: 17
 "actually not, I cannot finish my assns, and no one can help" size: 59
t?e greadest (?) , how are you?im fineactually not, I cannot finish my assns, and no one can help
